mapstruct-spi-accessor-naming/pom.xml

```diff
@@ -21,29 +21,28 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>org.mapstruct.examples.spi.accessornaming</groupId>
-    <artifactId>parent</artifactId>
+
+    <groupId>org.mapstruct.examples.spi</groupId>
```

Comment:
I added the `accessornaming` because we are already discussing more SPI's (e.g. templates, or selection)..